hopper/setup.py

```diff
@@ -631,6 +631,5 @@ def run(self):
         "torch",
         "einops",
         "packaging",
-        "ninja",
     ],
```

Comment:
Still need to put `ninja` under `setup_requires` (i.e. as a build time dependency) I think. See previous PR at #937.

```suggestion
    ],
        setup_requires=[
        "ninja",
```

Reply:
Thanks, I've moved packaging as well to align with your PR